README.md

```diff
@@ -1,3 +1,45 @@
-# Open Source Course Materials Template
+# OPEN EDUCATION (OPE) Test Book
 
-This template is to help get you started on your journey to creating Open Source educational content
\ No newline at end of file
+## About Test Book
+"Test Book" is based on the OPE course materials template. It contains various types of special cells that educators can re-use to author a similar OPE textbook. In general, the "Test Book" is aim to be a sampler book for the current and future courses. 
+
+Please notice that general cells are supported by *[Jupyter Book Plug-in](https://jupyterbook.org/en/stable/intro.html)*.
+
+When test against the "Test Book", if we can achieve the same results for both source code and visual output every time, we can conclude that the course materials are well-supported and are in good shape.
+
+The "Test Book" now mainly consist of 3 types of cells in Jupyter Notebook: static markdown cells with customized styles, interactive cells generated by custom codes, and slides and speaker note cells powered by RISE (a Plug-in to create slides within Jupyter Notebook).
+
+The current "Test Book" lives inside the content folder, and has been deployed on GitHub Pages.
+
+- Test Book textbook: https://xwshiba.github.io/ope/textbook
+- Test Book slides: https://xwshiba.github.io/ope/lecturenotes
+
+Please feel free to submit a pull request to add your customized cells from your book if you would like to include it into the base "Test Book".
+
+## Test Book Instruction
+
+- content folder - contains the special cells collected from various ope books.
+- baseline folder - contains the verified correct visual results we would like to achieve for every test run.
+- workflows folder - contains workflow files with relevant files for running GitHub Action. There are two workflows available: `cell_test` and `pdf_diff_test`.
+ - cell_test.yml - only test the source code of the Test Book, and preserve logs to the Actions tab.
+ - pdf_diff_test.yml - test both the source code and visual output of the Test Book. Logs for running the source code and pdfs for checking the visual output will be available to the Actions tab.
+
+## Test Book GitHub Action Instructions
+GitHub Action has a *[limited free plan](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)* for public repositories.
+
+Currently, the workflow will run on the push trigger to the `test-book` branch. The test will not exit if the content has any warnings or errors. Please view the log carefully.
+
+To detect visual difference, the test uses *[diff-pdf-visually](https://github.com/bgeron/diff-pdf-visually)* (MIT License). It will run the `pdf_diff_check.py` in the workflow folder. Since every test run will generate unique test run caches, the difference threshold is set at 50 instead of 100, to ignore the irrelevant caches.
+
+- To use Test Book GitHub Action, copy all the files in the `/workflow` folder to `/.github/workflows`.
```

Comment:
It might be better to change to `workflow` and `.github/workflows`, because `/workflow` indicates that it's an absolute path.